LambdaRuntimeDockerfiles/Images/net8/arm64/Dockerfile

```diff
@@ -3,40 +3,12 @@
 ARG ASPNET_VERSION=8.0.0-rc.1.23421.29
 ARG ASPNET_SHA512=ba8035da535cb3bffa720e962e6f9e0f88b36e1221b588f2a126ee4b43c02e4d8c27958017d29e5ab68121fab6a564fe0a27099c4103ee3d527f8554b4ab495e
 
-ARG ICU_VERSION=68.1
-ARG ICU_MD5=6a99b541ea01f271257b121a4433c7c0
-
 ARG LAMBDA_RUNTIME_NAME=dotnet8
-ARG AMAZON_LINUX=public.ecr.aws/lambda/provided:al2
+ARG AMAZON_LINUX=public.ecr.aws/lambda/provided:al2023-preview
 
 FROM $AMAZON_LINUX AS base
 
-FROM base AS builder-libicu
-WORKDIR /
-
-# Install depedencies to extract and build ICU library
-RUN yum install -d1 -y \
-    tar \
-    gzip \
-    make \
-    gcc-c++
-
-# Download, validate and extract ICU library
-# https://github.com/unicode-org/icu/releases/tag/release-68-1
-ARG ICU_VERSION
-ARG ICU_MD5
-RUN curl -SL https://github.com/unicode-org/icu/releases/download/release-${ICU_VERSION//./-}/icu4c-${ICU_VERSION//./_}-src.tgz -o icu-src.tgz \
-    && echo "$ICU_MD5  icu-src.tgz" | md5sum -c - \
-    && tar -xzf icu-src.tgz \
-    && rm icu-src.tgz
-
-# Build ICU library
-RUN mkdir /libicu
-WORKDIR /icu/source/
-RUN ./configure --prefix=/libicu \
-    && make \
-    && make install
-
+RUN dnf install libicu --assumeyes
```

Comment:
Do you recall why we were building libicu from source on AL2? Okay with the change, but just looking for context.

Reply:
If I remember correctly, the AL2 had LIBICU preinstalled but it had a version older than the one we needed for newer .NET versions. It's package manager had specific packages and versions available for installation, and LIBICU was not one of them. On AL2023, LIBICU is not preinstalled on the image, but `dnf` has version 67 available which works for the .NET versions we have. As soon as newer LIBICU versions are required, we will go back to installing from source.

Reply:
Done